docs/data/base/components/table-pagination/table-pagination.md

```diff
@@ -1,39 +1,43 @@
 ---
 product: base
-title: React Table pagination component
+title: Unstyled React Table Pagination component
 components: TablePaginationUnstyled
 githubLabel: 'component: TablePagination'
 packageName: '@mui/base'
 ---
 
 # Unstyled table pagination
 
-<p class="description">Table pagination lets users navigate through data in tables.</p>
+<p class="description">Table pagination is an interface tool for splitting up large amounts of data to make it easier for users to navigate.</p>
 
 {{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
 
-The TablePaginationUnstyled component controls two table properties: displayed page index and number of rows per page.
+The `TablePaginationUnstyled` component controls two properties of its parent table:
 
-By default it renders its internal components wrapped in a `tr` tag, so it's easy to insert it in a table's `tr`.
-This can be changed, if needed, by supplying the `component` or `components.Root` prop.
-This way you can place the pagination controls outside of the table.
+- displayed page index
+- number of rows per page
 
-## Basic TablePaginationUnstyled
+By default it renders its internal components wrapped in a `<tr>` tag, so it can be inserted into a table's `<tr>`.
```

Comment:
Should this be `<td>`?

```suggestion
By default it renders its internal components wrapped in a `<td>` tag, so it can be inserted into a table's `<tr>`.
```

Reply:
Yes, absolutely!